domains/transfer/Transfer.API/API/Program.cs

```diff
@@ -85,9 +92,66 @@
 
 builder.Services.AddLogging();
 
+
 builder.Services.AddTransfer(builder.Configuration);
 builder.Services.AddCvr();
 builder.Services.AddConnection();
+builder.Services.AddClaimServices();
+
+builder.Services.AddControllers(options => options.Filters.Add<AuditDotNetFilter>())
+    .AddJsonOptions(options =>
+        options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));
```

Comment:
There is a class named `JsonDefault` which adds the same converter (with a small difference). Why is `JsonDefault` not used here?

Reply:
To be removed

Reply:
Are you referring to the jsondefault from integration test?

Reply:
It is not the same. Closing